pages/common/cbt.md

```diff
@@ -0,0 +1,28 @@
+# cbt
+
+> Utility for reading data from Google Cloud's Bigtable.
+> More information: <https://cloud.google.com/bigtable/docs/cbt-reference>.
+
+- List tables in current project:
+
+`cbt ls`
+
+- Return count of rows in table:
+
+`cbt count {{table_name}}`
+
+- Read single row from a table with only 1 (most recent) cell revision per column:
+
+`cbt lookup {{table_name}} {{row_key}} cells-per-column={{1}}`
```

Comment:
```suggestion
`cbt lookup "{{table_name}}" "{{row_key}}" cells-per-column={{1}}`
```

Optional fix: `{{table_name}}` and `{{row_key}}` placeholders can be quoted if they can contain some special characters to POSIX-like shell.